CppExport/src/CodeComposite.cpp

```diff
@@ -41,6 +41,58 @@ void CodeComposite::addUnit(CodeUnit* unit)
 	unit->setComposite(this);
 }
 
+QString CodeComposite::relativePath(CodeComposite* other)
+{
+	QStringList thisName = name().split("/");
+	QStringList otherName = other->name().split("/");
+
+	if (name() != other->name())
+	{
+		while (thisName.first() == otherName.first())
+		{
+			thisName.takeFirst();
+			otherName.takeFirst();
+		}
+
+		int backSteps = thisName.size() - otherName.size();
+		for (auto i = 0; i < backSteps; i++) otherName.prepend("..");
+		return otherName.join("/");
+	}
+
+	return otherName.last();
```

Comment:
Plaese put this after the if and invert the condition of the if, to make this easier to read.